app/models/maintenance_tasks/task.rb

```diff
@@ -21,6 +21,16 @@ class NotFoundError < NameError; end
     # @api private
     class_attribute :collection_builder_strategy, default: NullCollectionBuilder.new
 
+    # The cursor_columns attribute is used to build the `ORDER BY` clause of the query for iteration.
+    # If the cursor_columns attribute is not set or is set to nil, the query is ordered by the primary key.
+    # If the values in the cursor columns change for the records in scope during an
+    # iteration, records may be skipped or yielded multiple times depending on the nature of the update and the
+    # cursor's values. If the values gets updated to greater values than the cursor's values, records will get yielded
```

Comment:
```suggestion
```
I think the rest is covered by the statement above. Let's link to job-iteration and provide an example of values here instead of going into more detail about the potentially unwanted behaviour.